test/helpers/kubectl.go

```diff
@@ -1315,10 +1323,12 @@ func (epMap *EndpointMap) GetPolicyStatus() map[models.EndpointPolicyEnabled]int
 	return result
 }
 
-// AreReady returns true if all Cilium endpoints are in 'ready' state
+// AreReady returns true if all Cilium endpoints are in 'ready' state and have
+// an identity different from 'reserved:init'.
 func (epMap *EndpointMap) AreReady() bool {
 	for _, ep := range *epMap {
-		if ep.Status.State != models.EndpointStateReady {
+		// Consider an endpoint with reserved identity 5 (reserved:init) as not ready.
+		if ep.Status.State != models.EndpointStateReady || ep.Status.Identity.ID == 5 {
```

Comment:
If the `5` here referred to the reserved:init identity by name, then the comment would be unnecessary.